scaleway/resource_baremetal_server.go

```diff
@@ -213,6 +222,20 @@ func resourceScalewayBaremetalServerCreate(ctx context.Context, d *schema.Resour
 		return diag.FromErr(err)
 	}
 
+	optionIDs := expandStrings(d.Get("option_ids").(*schema.Set).List())
+	if len(optionIDs) > 0 {
+		for i := range optionIDs {
+			_, err = baremetalAPI.AddOptionServer(&baremetal.AddOptionServerRequest{
+				Zone:     server.Zone,
+				ServerID: server.ID,
+				OptionID: expandID(optionIDs[i]),
```

Comment:
Why `ExpiresAt` is not considered? check [this](https://github.com/scaleway/scaleway-sdk-go/blob/master/api/baremetal/v1/baremetal_sdk.go#L1825)

Reply:
Added